scripts/split_repo_migration/delete-everything-split-repo.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script was written as part of a project to migrate
+# split repositories to a mono repository google-cloud-python.
+# This script will delete all files in a split repository and
+# update the README to indicate that the repository is archived.
+
+# `-e` enables the script to automatically fail when a command fails
+set -e
+
+# Ensure that both arguments are provided
+# Args:
+#   split-repo-name - The name of the split repository. For example `python-access-approval`.
+#   artifact-name - The name of the artifact on PyPI. For example `google-cloud-access-approval`.
+if [ $# -lt 2 ]
+then
+  echo "Usage: $0 <split-repo-name> <target-path>"
+  exit 1
+fi
+
+# The name of the split repository. (e.g. python-access-approval)
+SPLIT_REPO=$1
+# The name of the artifact on PyPI. (e.g. google-cloud-access-approval)
+ARTIFACT_NAME=$2
+
+# Quick clean up in case a folder already exists in /tmp/
+rm -rf "/tmp/${SPLIT_REPO}"
+
+# Clone the split repo into /tmp/<name of split repo>
+git clone "git@github.com:googleapis/${SPLIT_REPO}.git" "/tmp/${SPLIT_REPO}"
+
+# cd into the split repo clone
+cd "/tmp/${SPLIT_REPO}"
+
+# Create a git branch
+git checkout -b 'delete-library'
+
+# Quick check to make sure there are no handwritten samples. 
+# Handwritten samples should be migrated to python-docs-samples before repository code is deleted.
+# Exclude the samples/generated_samples/* directory which contains autogenerated samples.
+NUM_PY_SAMPLES_FILES=$(find samples -type f -name "*.py" ! -path "samples/generated_samples/*" | wc -l)
+
+# Fail if there are handwritten samples.
+if [[ $NUM_PY_SAMPLES_FILES != "0" ]]; then
```

Comment:
Technically, when comparing numbers we should use `[[ ... --eq ... ]]` or `(( ... == ... ))`. Probably the output in these checks is constrained enough that string comparison is fine.
https://tldp.org/LDP/abs/html/comparison-ops.html
https://tldp.org/LDP/abs/html/dblparens.html

Reply:
Good catch! Thanks for the helpful links!

Reply:
Fixed in 4bf69d903